src/ngraph/pass/constant_folding.hpp

```diff
@@ -50,6 +50,7 @@ class NGRAPH_API ngraph::pass::ConstantFolding : public ngraph::pass::GraphRewri
         LOGICAL_REDUCTION,
         CONCAT,
         GATHER,
+        SCATTER,
```

Comment:
May be rename this to SCATTER_ELEMENTS_UPDATE?

Reply:
I wonder if we would add more operators `ScatterXXX` could we put all of them into this category?